Source/Microsoft.Teams.Apps.FAQPlusPlus/Cards/MessagingExtensionQnaCard.cs

```diff
@@ -31,14 +31,20 @@ public static class MessagingExtensionQnaCard
         private const string GoToOriginalThreadUrl = "https://teams.microsoft.com/l/message/";
 
         /// <summary>
-        /// Date format.
+        /// Short date and time format to support adaptive card text feature.
         /// </summary>
-        private const string DateFormat = "{{{{DATE({0}, SHORT)}}}} {{{{TIME({1})}}}}";
+        /// <remarks>
+        /// refer adaptive card text feature https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/text-features#datetime-formatting-and-localization
+        /// </remarks>
+        private const string AdaptiveCardShortDateTimeFormat = "{{{{DATE({0}, SHORT)}}}} {{{{TIME({1})}}}}";
```

Comment:
[Not blocking] Change this to use the same placeholder {0}, as logically there is only 1 parameter here. Currently you are passing 2.